src/content/docs/configuration/providers.mdx

```diff
@@ -0,0 +1,96 @@
+---
+title: Providers
+description: Understand Providers and configure new Targets.
+sidebar:
+  label: Providers
+---
+
+Daytona abstracts the management and deployment of Workspaces into 2 related concepts:
+
+1.  __Provider__  
+    A platform capable of deploying Workspaces on.
```

Comment:
`An application capable....` Or maybe `plugin`. `platform` gives the Provider too much weight.

Reply:
Rephrased in [92c92a0](https://github.com/daytonaio/docs/pull/24/commits/92c92a08286b290cdd0c774c7ac01248129d7f1a).